vllm/entrypoints/openai/serving_completion.py

```diff
@@ -137,14 +137,14 @@ async def create_completion(self, request: CompletionRequest,
 
             for i, prompt in enumerate(prompts):
                 if prompt_is_tokens:
-                    input_ids = self._validate_prompt_and_tokenize(
+                    input_ids, input_text = self._validate_prompt_and_tokenize(
```

Comment:
How about naming them `prompt_ids` and  `prompt_text`? (for better code consistency with the `serving_chat.py`)